features/index.feature

```diff
@@ -0,0 +1,44 @@
+Feature:
+   As a User|
+   In order to navigate index.erb
+   I want to see a menu
```

Comment:
This user story doesn't really make sense. What is the *business objective* of this feature? Perhaps something like "In order to move through the site I need to have an index page and navigation menu"?